### PR TITLE
CRYP-95: add basic settings page and navigation

### PR DESCRIPTION
**Pull request by jason-gerard**
JIRA: https://cryptify.atlassian.net/browse/CRYP-95

## Changes
- Add settings screen
- Add screen to navigation
- Add header styling for blank page

## Screenshots
image

### CODE REVIEW

**Review by Andre-Ibrahim — APPROVED** [review]
lgtm

**Comment by domsec** [comment]
There shouldn't be a grey line present.

File: packages/client/navigation/index.tsx [hunk 1]
```diff
@@ -65,6 +66,20 @@ function BottomTabNavigator() {
                     tabBarIcon: ({ color }) => <TabBarIcon name="code" color={color} />,
                 }}
             />
+            <BottomTab.Screen
+                name="SettingsScreen"
+                component={SettingsScreen}
+                options={{
+                    title: "Settings",
```
Comment (domsec) [inline packages/client/navigation/index.tsx:73]: Is it possible to use the Text component in order to take advantage of the custom theme?

**Comment by jason-gerard** [comment]
> There shouldn't be a grey line present.

I can't even see it lol, i'll try to change the drop shadow of the header i think it might work.

File: packages/client/navigation/index.tsx [hunk 2]
```diff
@@ -65,6 +66,20 @@ function BottomTabNavigator() {
                     tabBarIcon: ({ color }) => <TabBarIcon name="code" color={color} />,
```
Comment (domsec) [inline packages/client/navigation/index.tsx:66]: should be a menu icon

Comment (jason-gerard) [reply packages/client/navigation/index.tsx:66]: Yea this should all get taken care of in the user story for the navbar. ideally he will make some component for future use with all the styling working.

File: packages/client/navigation/index.tsx (hunk 1 above)
Comment (jason-gerard) [reply packages/client/navigation/index.tsx:73]: Im reading now, it looks like there should be a way, let me test it out in a bit

File: packages/client/navigation/index.tsx (hunk 2 above)
Comment (domsec) [reply packages/client/navigation/index.tsx:66]: Perfect.

File: packages/client/navigation/index.tsx (hunk 1 above)
Comment (jason-gerard) [reply packages/client/navigation/index.tsx:73]: Ok so basically would have to make a completely new Text component anyways so not sure if its worth it

**Pull request merged**